src/discretization/reconstruction.cpp

```diff
@@ -307,7 +307,26 @@ void ReconstructionPolynomial::evalPointBasisDerivatives(uint8_t degree, uint8_t
 /*!
  * Evaluates the values of the basis for cell reconstruction.
  *
- * The method works only for ElementType::Voxel and ElementType::Pixel.
+ * Basis functions are evaluated as the volume average of the Taylor series
+ * expansion coefficients:
+ *
+ *              /
+ *      (1 / V) | (csi_0 * U_0 + csi_1 * U_1 + csi_2 * U_2 + ... ) dV
+ *              /
+ *
+ * where:
+ *
+ *   - csi_0 = 1;
+ *   - csi_1 = x;
+ *   - csi_2 = y;
+ *   - csi_3 = z;
+ *   - csi_4 = 0.5 * (x - x_0)^2;
+ *   - csi_5 = 0.5 * (y - y_0)^2;
+ *   - csi_6 = 0.5 * (z - z_0)^2;
+ *   - ...;
+ *
+ * Polynomials of degree 0 and 1 are supported on all type of cells, for pixels
+ * and voxels also degree 3 is supported.
```

Comment:
There's a discrepancy between doc and code. Max supported degree in code is 2, while doc says 3.

Reply:
I pushed a fix